### PR TITLE
Enable read-session caching by default for faster Spark planning

### PR DESCRIPTION
**Pull request by vishalkarve15**

### CODE REVIEW

**Review by anoopj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -860,6 +860,15 @@ word-break:break-word
      </td>
      <td>Read</td>
    </tr>
+  <tr>
+     <td><code>enableReadSessionCaching</code>
+     </td>
+     <td>Boolean config to disable read-session caching. Caches BigQuery read-sessions to allow for faster Spark query planning.
```
Comment (anoopj) [inline README.md:866]: s/read-sessions/read sessions/?

File: CHANGES.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 * GAX has been upgraded to version 2.35.0
 * gRPC has been upgraded to version 1.58.0
 * Protocol Buffers has been upgraded to version 3.24.4
+* Enable read session caching by default for faster Spark planning
```
Comment (davidrabinowitz) [inline CHANGES.md:15]: Please change to `PR #1109:  Enable read session caching by default for faster Spark planning` and place after line 8 (to keep the PRs ordered)

File: README.md [hunk 3]
```diff
@@ -860,6 +860,15 @@ word-break:break-word
      </td>
      <td>Read</td>
    </tr>
+  <tr>
```
Comment (davidrabinowitz) [inline README.md:863]: Please remove - as this is the first documentation of the flag it should appear in README-template.md only (also,  it's current default vale is `false`)

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]

**Review by davidrabinowitz — APPROVED** [review]

**Pull request merged**